.github/workflows/build-and-push-binaries.yml

```diff
@@ -31,11 +31,20 @@ jobs:
             shell: bash
           - os: windows-latest
             shell: cmd
+        models:
+          - provider: ChatIBMGenAI
+            model_id: meta-llama/llama-3-70b-instruct
+            max_new_tokens: 2048
     runs-on: ${{ matrix.runs_on.os }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "temurin"
```

Comment:
why this one and not like openjdk? or am I missing something?

Reply:
@shawn-hurley openjdk isn't available with this action, but temurin is basically the Eclipse version and works